### PR TITLE
Update to 5.7.0

### PR DESCRIPTION
**Pull request by pearu**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: .ci_support/linux_64_cuda_compiler_version11.0.yaml [hunk 1]
```diff
@@ -42,6 +42,11 @@ librdkafka:
 - '1.6'
 ncurses:
 - '6.2'
+numpy:
+- '1.18'
+- '1.18'
+- '1.18'
```
Comment (vnlitvinov) [inline .ci_support/linux_64_cuda_compiler_version11.0.yaml:48]: I wonder why do you need three equal entries?..

Comment (pearu) [reply .ci_support/linux_64_cuda_compiler_version11.0.yaml:48]: I guess these three entries correspond to python versions 3.6, 3.7, and 3.8, and the subsequent entry 1.19 corresponds to python version 3.9.

**Comment by pearu** [comment]
@andrewseidl @jclay @xmnlab please review. I'll squash/merge in two days unless there will be any review activity.

**Comment by Garra1980** [comment]
Shouldn't arrow version be upgraded to 3. or even 4.?

**Comment by pearu** [comment]
> Shouldn't arrow version be upgraded to 3. or even 4.?

AFAIK, omniscidb is not arrow 3. ready.

**Comment by Garra1980** [comment]
I just updated omnisci deps and see that libarrow3.0.0 is used

**Comment by pearu** [comment]
Indeed. I'll do a follow-up PR for arrow 3.

For the record, when using arrow 4, the build fails with:
```
omniscidb-5.7.0/DataMgr/ForeignStorage/ArrowForeignStorage.cpp:1006:86: error: no matching function for call to 'arrow::csv::TableReader::Make(arrow::MemoryPool*&, std::shared_ptr<arrow::io::ReadableFile>&, arrow::csv::ReadOptions&, arrow::csv::ParseOptions&, arrow::csv::ConvertOptions&)'
```

**Comment by Garra1980** [comment]
Ah, right, I saw this sometime ago. Sure, let's stick to 3.0 for now

**Comment by pearu** [comment]
https://github.com/conda-forge/omniscidb-feedstock/pull/50

**Pull request merged**